src/Administration/Resources/app/administration/src/module/sw-cms/snippet/de-DE.json

```diff
@@ -109,22 +111,22 @@
         "buttonMappingRemoveAction": "Datenzuordnung entfernen",
         "mappingPreview": "Datenzuordnung:",
         "mappingEmptyPreview": "Für diese Datenzuordnung liegen leider keine Vorschaudaten vor.",
-        "addBlocks": "Füge Blöcke per Drag & Drop hinzu",
+        "addBlocks": "Füge Blöcke per Drag-and-drop hinzu",
```

Comment:
seems to be valid https://de.wikipedia.org/wiki/Drag_and_Drop :thinking:

Reply:
https://www.duden.de/rechtschreibung/Drag_and_drop 🤔

Reply:
In my opinion, we should leave the old variant. Drag & Drop as such is a defined term, which isn't wrong in the german language. The & symbol doesn't look like a foreign element in the sentence, while "and" sure does.

Reply:
Würde das jetzt auch erstmal nicht anfassen wollen. Bisher ist Drag & Drop in einheitlicher Verwendung, wenigstens in den Snippets. Drag-and-drop finde ich gar nicht schön und es führt zusätzlich eine neue Uneinheitlichkeit ein (Drag großgeschrieben, drop kleingeschrieben)

Reply:
reverted to "Drag & Drop"